draft-ietf-quic-transport.md

```diff
@@ -1069,6 +1069,17 @@ to cease using the connection IDs when requested can result in connection
 failures, as the issuing endpoint might be unable to continue using the
 connection IDs with the active connection.
 
+An endpoint MAY elect to limit the number of outstanding RETIRE_CONNECTION_ID
+frames to bound the necessary state. In order to allow a peer to retire all
+previously issued connection IDs, this limit SHOULD be at least the
+active_connection_id_limit. Within this limit, endpoints SHOULD prioritize
```

Comment:
I don't think this last sentence is necessary, so I'd be inclined towards removing it.

Reply:
I'm not sure which one you're referring to.